Structure_oM/SurfaceProperties/Reinforcement/PanelRebarIntent.cs

```diff
@@ -0,0 +1,45 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base;
+using BH.oM.Quantities.Attributes;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+
+namespace BH.oM.Structure.SurfaceProperties.Reinforcement
+{
+    [Description("A collection of PanelReinforcement objects that represent the rebar intent. \n" +
+        "This is stored as layers of reinforcement amongst regions within the Panel.")]
+    public class PanelRebarIntent : BHoMObject, IFragment, IRebarIntent
+    {
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+
+        [Description("A collection of PanelReinforcement objects.")]
+        public virtual List<PanelReinforcement> PanelReinforcement { get; set; }
```

Comment:
I would initialise the list here to avoid unexpected nulls.
```suggestion
        public virtual List<PanelReinforcement> PanelReinforcement { get; set; } = new List<PanelReinforcement>();
```